src/pivotal-ui/components/viewport-panes/viewport-panes.scss

```diff
@@ -0,0 +1,63 @@
+@import "../pui-variables";
+
+/*doc
+---
+title: Viewport Pane
+name: viewport_pane
+categories:
+ - css_utilities_panes
+ - css_all
+---
+
+<code class="pam">
+<i class="fa fa-download" alt="Install the Component">
+npm install @npmcorp/pui-css-viewport-panes --save
+</i>
+</code>
+
+Viewport panes are type of [pane](/pane.html) that have a height that is based upon
+the height of your viewport.
+
+You can add the `.viewport-pane` class to the top pane of a layout.
+
+```html_example
+<header>
+  <h1>This is the header</h1>
+</header>
+<div class="pane bg-dark-2 viewport-pane">
+  <div class="container bg-glow">
+    <h1 class="type-neutral-11">This is a pane</h1>
+  </div>
+</div>
+
+```
+
+When following a transparent header, the viewport-pane class will allow the pane's color
+to come through
+
+```
+<header class="hero-transparent">
+  <h1>This is the header, you can see the pane below me, through this</h1>
+</header>
+<div class="pane bg-dark-2 viewport-pane">
+  <div class="container bg-glow">
+    <h1 class="type-neutral-11">This is a pane</h1>
+  </div>
+</div>
+```
+
+*/
+
+.viewport-pane {
+  height: calc(100vh - $header-height-max-sm);
+}
+
+.hero-transparent + .viewport-pane {
+  box-sizing: content-box;
+}
+
+@media (min-width: $screen-sm-min) {
+  .viewport-pane {
+    height: calc(80vh - $header-height-max-lg);
```

Comment:
also, I think this should be min-height. In the unfortunate case where we have too much content for on screen height, we don't want things to get cropped.